README.adoc

```diff
@@ -324,31 +327,54 @@ Next, use the following `startContainers.sh` script to start the application in
 
 include::{common-includes}/os-tabs.adoc[]
 
-[.tab_content.mac_section.linux_section]
+
+[.tab_content.windows_section]
 --
 [role='command']
 ```
-./scripts/startContainers.sh
+.\scripts\startContainers.bat
 ```
 --
 
-[.tab_content.windows_section]
+[.tab_content.mac_section.linux_section]
 --
 [role='command']
 ```
-.\scripts\startContainers.bat
+./scripts/startContainers.sh
 ```
 --
 This script creates a network for the containers to communicate with each other. It
 also creates containers for Kafka, Zookeeper, the `frontend` service, the `bff` service , and three
 instances of the `system` service.
 
+// Static guide instruction
+ifndef::cloud-hosted[]
+The application might take some time to get ready.
+Visit the http://localhost:9084/health[^] URL to confirm that the `bff` microservice is up and running.
+
 Once your application is up and running, open your browser and check out your
-service by visiting http://localhost:9080[http://localhost:9080]. The
-application might take some time to get ready. The latest version of most
-modern web browsers supports Server-Sent Events. The exception is
-Internet Explorer, which does not support SSE.
+service by visiting http://localhost:9080[http://localhost:9080]. 
+endif::[]
 
+// Cloud hosted guide instruction
+ifdef::cloud-hosted[]
+The application might take some time to get ready.
+Run the following command to confirm that the **bff** microservice is up and running:
+```
+curl http://localhost:9084/health
+```
+{: codeblock}
+
+Once your application is up and running, use the following command to get the url.
+Open your browser and check out your service by visiting the url. 
```

Comment:
Open your browser and check out your service by going to the URL that the command returns